lib/Models/Catalog/Esri/ArcGisMapServerCatalogItem.ts

```diff
@@ -540,6 +541,8 @@ export default class ArcGisMapServerCatalogItem extends UrlMixin(
         false
       );
 
+      arraysAreEqual;
```

Comment:
What is it for?

Reply:
Good question! Not sure why I left this in - I'll remove it!